tasks/browser.mjs

```diff
@@ -0,0 +1,25 @@
+import puppeteer from 'puppeteer'
+
+import configPuppeteer from '../puppeteer.config.js'
+
+/**
+ * Download Puppeteer browser
+ */
+export async function download () {
+  const fetcher = puppeteer.createBrowserFetcher({
+    path: configPuppeteer.cacheDirectory
+  })
+
+  // Downloaded versions
+  const versions = fetcher.localRevisions()
+
+  // Download latest browser (unless cached)
+  if (!versions.includes(puppeteer.browserRevision)) {
+    await fetcher.download(puppeteer.browserRevision)
+
+    // Remove outdated browser versions
+    for (const version of versions) {
+      await fetcher.remove(version)
+    }
+  }
+}
```

Comment:
*question*: Could we use Puppeteer's public installation script instead of maintaining our own? Looks like [`puppeteer/install.js`](https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer/install.js) is meant for doing the browser download? I'm a bit warry of having to keep our own script up to date when Puppeteer already maintains one 😄

Reply:
@romaricpascal Yeah this is what we were using on `npm ci` (behind the scenes) but it gets cached

Stopped it running with `{ skipDownload: true }` but yep happy to see if I can import it

Leave it with me

Reply:
Ahh it was a nice idea but it just [logs](https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer/src/node/install.ts#L40):

```console
**INFO** Skipping browser download as instructed.
```

Only other idea is to flip `PUPPETEER_SKIP_DOWNLOAD` **true** or **false** in the GitHub workflows?